mne/io/brainvision/brainvision.py

```diff
@@ -55,7 +55,7 @@ class RawBrainVision(_BaseRaw):
         triggers will be ignored. Default is 0 for backwards compatibility, but
         typically another value or None will be necessary.
     event_id : dict | None
-        The id of the event to consider. If None (default),
+        The id of special events to consider. If None (default),
```

Comment:
if you use event_id, does it include in addition to the `SXXX` or is it in lieu of?